corehq/apps/sso/tasks.py

```diff
@@ -129,10 +126,9 @@ def auto_deactivate_removed_sso_users():
     for idp in IdentityProvider.objects.filter(
         enable_user_deactivation=True,
         idp_type=IdentityProviderType.ENTRA_ID,
-        login_enforcement_type=LoginEnforcementType.GLOBAL,
     ).all():
         try:
-            usernames_in_idp = convert_emails_to_lowercase(idp.get_all_usernames_of_the_idp())
+            usernames_in_idp = set(convert_emails_to_lowercase(idp.get_all_usernames_of_the_idp()))
```

Comment:
`get_all_usernames_of_the_idp` and `get_webuser_names_goverened_by_idp` are pretty confusing names at a glance

suggestions:
- `get_all_usernames_of_the_idp` renamed to `get_remote_member_usernames`
- `get_webuser_names_govered_by_idp` renamed to `get_local_member_usernames`